src/components/Notification.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <!-- content wrapper  -->
+  <div
+    class="flex flex-row h-24 w-full mx-auto bg-white shadow-md overflow-hidden m-5"
+  >
+
+    <!-- info 1    -->
+    <div class="grow">
+      <div class="flex flex-row mt-2 ml-2 ">
+        <div class="pr-2 font-bold">
+
+          <!--comment-->
+          <p v-if="theNotification.notificationType === 'comment'">New comment for</p>
+          <p v-else-if="theNotification.notificationType === 'favourite'">{{theNotification.userName}} just saved</p>
+          <p v-else-if="theNotification.notificationType === 'like'">{{theNotification.userName}} just liked</p>
+        </div>
+
+        <div class="text-primary-orange font-bold">
+          <!--comment && favourite-->
+          <p>{{showText(theNotification.postName)}}</p>
+        </div>
+      </div>
+
+      <div class="flex flex-row mt-2 ml-2 text-sm">
+        <div class="pr-2 ">
+          <!--comment-->
+          <p v-if="theNotification.notificationType === 'comment'">{{theNotification.userName}} wrote</p>
+        </div>
+
+        <div class="">
+          <!--comment-->
+          <p v-if="theNotification.notificationType === 'comment'">{{showText(theNotification.text)}}</p>
+        </div>
+      </div>
+
+      <div class="flex flex-row mt-2 ml-2 text-xs text-zinc-400">
+        <div class="">
+          <!--comment && favourite -->
+          <p class="">{{formatDate(theNotification.date)}}</p>
+        </div>
+
+      </div>
+
+    </div>
+
+    <!-- take all of the left space - Buttons -->
+    <div class="flex flex-row items-center">
+      <div  @click="$emit('redirect')" class="lg:pr-2 xl:pr-4 2xl:8 hover">
+        <svg
```

Comment:
no no no, we don't need to do it like that, check my other components, just import an icon from heroicons, its so much easier to use and we stay consistent :) also those are huuuuge, I think we use size 5 everywhere else
<img width="624" alt="Screenshot 2022-05-23 at 20 29 07" src="https://user-images.githubusercontent.com/71661786/169883899-6ac404d8-309e-455d-82ff-9209e55ca117.png">
<img width="624" alt="Screenshot 2022-05-23 at 20 29 14" src="https://user-images.githubusercontent.com/71661786/169883924-20f8092a-4c8d-458a-a36b-529213833720.png">